manifests/m/MasterPackager/MasterPackager/22.1.8131.0/MasterPackager.MasterPackager.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created with YamlCreate.ps1 v2.0.7 using InputObject 🤖 $debug=QUSU-7-2-2
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: MasterPackager.MasterPackager
+PackageVersion: 22.1.8131.0
+MinimumOSVersion: 10.0.0.0
+InstallerType: wix
+Scope: machine
+Installers:
+- Architecture: x64
+  InstallerUrl: https://www.masterpackager.com/installer/public/standard/masterpackager_22.1.8131.0.msi
```

```suggestion
  InstallerUrl: https://www.masterpackager.com/installer/masterpackager_22.1.8131.0.msi
```
Have you tried this?